Platon/backend/app/auth_system/views.py

```diff
@@ -100,6 +105,225 @@ def post(user_id,self):
         else:
             return make_response(jsonify({'error' : 'Write new password twice'}),400)
 
+
+@auth_system_ns.route("/user")
+class UserAPI(Resource):
+    '''
+    This class is a RESTful API for "User" model.
+    You can find the endpoints for creating, reading, updating and deleting a user below.
+    '''
+
+    # GET request
+    @api.expect(get_user_parser)
+    @api.doc(responses={
+                200: "User has been found.",
+                400: "Missing data fields or invalid data.",
+                404: "The user is not found.",
+                500: "The server is not connected to the database.",
+            })
+    def get(self):
+        # Parses the form data.
+        form = GetUserForm(request.args)
+        
+        # Checks whether the data is in valid form.
+        # If yes, starts processing the data.
+        # If not, an error is raised.
+        if form.validate():
+            # Tries to connect to the database.
+            # If it fails, an error is raised.
+            try:
+                existing_user = User.query.filter_by(id=form.user_id.data).first()
+            except:
+                return make_response(jsonify({"error" : "The server is not connected to the database."}), 500)
+            else:
+                # Checks whether there is an already existing user in the database with the given user ID.
+                # If yes, user information is returned.
+                # If not, an error is raised.
+                if existing_user is not None:
+                    # DO NOT FORGET TO WRITE CODE FOR PRIVATE ACCOUNTS
```

Comment:
For private accounts, get method should not return user data. It would be good to have a control mechanism for that. 

My suggestion would be using auth_token. In this case, requests will contain token in the header. Token contains the current user's ID. Add the login_required property which outputs the user_id. By that ID, you can check if the current user is in the followers list of other user.

Reply:
@cakmadam98 Thank you for the suggestion, I agree with you. That was the reason why I added `# DO NOT FORGET TO WRITE CODE FOR PRIVATE ACCOUNTS` line to the code, as the "GET" should have more complex behavior. I hope to update it soon.